crypto/evp_extra/p_kem.c

```diff
@@ -32,6 +32,44 @@ static void pkey_kem_cleanup(EVP_PKEY_CTX *ctx) {
   OPENSSL_free(ctx->data);
 }
 
+static int pkey_kem_keygen_deterministic(EVP_PKEY_CTX *ctx,
+                                         EVP_PKEY *pkey,
+                                         const uint8_t *seed,
+                                         size_t *seed_len) {
+  KEM_PKEY_CTX *dctx = ctx->data;
+  const KEM *kem = dctx->kem;
```

Comment:
```suggestion
  GUARD_PTR(ctx);
  KEM_PKEY_CTX *dctx = ctx->data;
  GUARD_PTR(dctx);
  const KEM *kem = dctx->kem;
```

Reply:
Thank you, added in [6aa7b7d](https://github.com/aws/aws-lc/pull/1578/commits/6aa7b7d2e5b3f035ad8fae576f529744a8dd1f04).